src/binding.cc

```diff
@@ -1565,6 +1565,16 @@ void v8__Function__SetName(const v8::Function& self, const v8::String& name) {
   return ptr_to_local(&self)->SetName(ptr_to_local(&name));
 }
 
+int v8__Function__GetScriptColumnNumber(const v8::Function& self) {
+  int ret = ptr_to_local(&self)->GetScriptColumnNumber();
+  return ret == v8::Function::kLineOffsetNotFound ? -1 : ret;
```

Comment:
```suggestion
  return ptr_to_local(&self)->GetScriptColumnNumber();
```
kLineOffsetNotFound == -1 so the ternary is somewhat superfluous.